azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/implementation/WebAppsImpl.java

```diff
@@ -63,7 +63,7 @@ public WebApp call(SiteConfigResourceInner siteConfigResourceInner, SiteLogsConf
             @Override
             public Observable<WebApp> call(final SiteInner siteInner) {
                 if (siteInner == null) {
-                    return null;
```

Comment:
Should be Observable.complete()

Reply:
Seems no such thing as `Observable.complete()`?

`null` is actually OK for rxjava 1.x, which is used here. rxjava 2.x would reject null.